ginga/aggw/Plot.py

```diff
@@ -7,8 +7,12 @@
 # This is open-source software licensed under a BSD license.
 # Please see the file LICENSE.txt for details.
 #
-from matplotlib.backends.backend_agg import FigureCanvasAgg \
-     as FigureCanvas
+from __future__ import absolute_import
+
+from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
+
+from . import Widgets
```

Comment:
Note: I am really not sure about this one, but thought it is what PlotWidget below is attempting to inherit from?